### PR TITLE
Added error message when failed to process

### PR DESCRIPTION
**Pull request by yoavkatz**
value in instrance operator and augmentors

### CODE REVIEW

**Review by matanor — CHANGES_REQUESTED** [review]
please see  a few comments

File: src/unitxt/test_utils/operators.py [hunk 1]
```diff
@@ -21,8 +21,33 @@ def apply_operator(operator: StreamingOperator, inputs: List[dict], return_multi
...
+
+
 def test_operator(
-    operator: StreamingOperator, inputs: List[dict], targets: List[dict], tester=None, sort_outputs_by=None
+    operator: StreamingOperator,
+    inputs: List[dict],
+    targets: List[dict],
+    tester=None,
+    sort_outputs_by=None,
+    exception_text=None,
```
Comment (matanor) [inline src/unitxt/test_utils/operators.py:50]: why is exception_text added? i don't see its used?

**Review by matanor — APPROVED** [review]
LGTM, left one small comment

File: src/unitxt/test_utils/operators.py (hunk 1 above)
Comment (yoavkatz) [reply src/unitxt/test_utils/operators.py:50]: I'll remove.  It was added by mistake.

**Pull request merged**